Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -1620,23 +1620,22 @@ protected String getTitleDefinition(String projectTitle, String docType) throws
     }
 
     /**
-     * Conditions:
-     * isDocType.equals("") && isNotDocType.equals("")
-     *     - nothing was specified
-     * isNotDocType.equals("") && StringUtils.containsIgnoreCase(isDocType, docType)
-     *     - only duty was specified
-     * isDocType.equals("") && !StringUtils.containsIgnoreCase(isNotDocType, docType)
-     *    - only may not was specified
-     * !isDocType.equals("") && !isNotDocType.equals("") && StringUtils.containsIgnoreCase(isDocType, docType)
-     *                 && !StringUtils.containsIgnoreCase(isNotDocType, docType)
-     *    - both were specified
+     * Conditions: isDocType.equals("") && isNotDocType.equals("") - nothing was
+     * specified isNotDocType.equals("") &&
+     * StringUtils.containsIgnoreCase(isDocType, docType) - only duty was
+     * specified isDocType.equals("") &&
+     * !StringUtils.containsIgnoreCase(isNotDocType, docType) - only may not was
+     * specified !isDocType.equals("") && !isNotDocType.equals("") &&
+     * StringUtils.containsIgnoreCase(isDocType, docType) &&
+     * !StringUtils.containsIgnoreCase(isNotDocType, docType) - both were
+     * specified
      */
     private String findTitleDefinition(String title, String docType, String isDocType, String isNotDocType) {
         if ((isDocType.equals("")
                 && (isNotDocType.equals("") || !StringUtils.containsIgnoreCase(isNotDocType, docType)))
                 || (!isDocType.equals("") && !isNotDocType.equals("")
-                && StringUtils.containsIgnoreCase(isDocType, docType)
-                && !StringUtils.containsIgnoreCase(isNotDocType, docType))
+                        && StringUtils.containsIgnoreCase(isDocType, docType)
```

Comment:
This format doesn't make sense. Why previous line is not moved only this 2?

Reply:
Because this is a block enclosed in `()`. So the inner **and** is more indented than the outer **or**. By the way, this is caused by the provided code formatter settings. I don't format code manually.

Reply:
This is the code formatter on its own. I think because of the additional level of parentheses. It makes some sense.